proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,121 @@
+4. Expulsion may occur due to serious violations or non-payment of fees over 12 months. Members must be given the opportunity to justify their actions before expulsion.
+5. Members leaving the Association or if the Association is liquidated do not have a right to reimbursement for contributed assets.
+
+## 5. Organs of the Association
+
+- The General Assembly
+- The Board
+
+## 6. General Assembly
+
+1. The General Assembly consists of all active members, each with one vote.
+2. It takes place at least once a year, with invitations sent at least six weeks prior.
+3. An extraordinary General Assembly may be called if requested by at least 20% of members.
+4. The General Assembly has a quorum if the invitation was submitted in proper form.
+5. A ¾ majority of votes is required to amend the Articles of Association or decide the liquidation of the Association.
+6. Active members unable to attend may be represented by another member with written authorization.
+
+## 7. Duties of the General Assembly
+
+1. The General Assembly is the highest decision-making organ and is responsible for all duties unless otherwise assigned.
+2. It elects the Board and can deselect board members.
+3. It decides on member expulsion objections.
+4. It receives the Board’s annual report and auditor’s report, and discharges the Board’s responsibilities.
+5. It decides on amendments to the Articles of Association and liquidation of the Association.
+6. It presents annual accounts and reports, and appoints auditors.
+7. It decides on purchases, sales, investments, and raising loans.
+
+## 8. The Board
+
+nsists of the Chairman, the Second Chairman, and the Treasurer. 
+
+1. The Board consists of the Chairman, the Second Chairman, and the Treasurer elected for a 3-year term.
+2. The Board’s duties and meeting procedures are stipulated in the by-laws.
+3. The Board decides on all matters unless a General Assembly decision is required.
+4. Each Board member has the right to represent the Association.
+5. Provisional board members are appointed in case of a vacancy until the next General Assembly.
+6. The Board can appoint a full-time director to manage current affairs.
+7. The director participates in General Assemblies and Board meetings.
+8. The Board can amend the Articles of Association for formal reasons as required by authorities.
+
+## 9. By-Laws
+
+The by-laws govern all details of the Association’s activities not covered in the Articles of Association. The Board implements the by-laws following member approval.
+
+## 10. Meeting Minutes
+
+All decisions by the Board and General Assembly are documented in writing and available for members’ perusal.
+
+## 11. Labour Agreements
+
+The Board regulates remuneration for full-time employees in the by-laws.
+
+## 13. Financing of the Association
+
+1. The Association is funded through membership fees, public contributions, donations, and other financial contributions.
```

Comment:
"other financial contributions" is kind of handwavy, does this need to be defined further or do you think this suffices?

Also, should we add here that the association is non-profit and any revenue (e.g. from selling merchandise) shall be used in accordance with article 3.2.

Reply:
I have no idea. The good thing is that all this has been started from the original KDE.eV Satzung (see git log).
That dos not mean we can't improve it. So go ahead ..